util/container/Dockerfile

```diff
@@ -54,6 +64,15 @@ RUN add-apt-repository -y ppa:deadsnakes/ppa && \
   ln -fs /usr/bin/python3.10 /usr/bin/python3 && \
   ln -fs /bin/python3.10 /bin/python3
 
+# Get prebuilt CVA6 gcc toolchain
+ENV CVA6_GCC_URL=https://github.com/KULeuven-MICAS/llvm-build/releases/download/test-7/custom_toolchain_build.tar.gz
```

Comment:
test-7 seems like a random identifier?
Also, why do we need to build this ourselves?
There are plenty other sources to get builds from, no? Such as : https://github.com/riscv-collab/riscv-gnu-toolchain/releases